src/wp-includes/formatting.php

```diff
@@ -2112,36 +2112,39 @@ function sanitize_file_name( $filename ) {
 }
 
 /**
- * Returns true if the string contains no more than one unicode
- * script, and false if it contains two or more. This only considers
- * alphabetic characters.
+ * Validates that a string contains only characters from a single unicode script.
```

Comment:
I unwaffled this: basically just put the "what" before the "why" to make it easier (for me) to follow.

Reply:
It was like that for a reason, actually.

My distant past includes [the qt documentation](https://rant.gulbrandsen.priv.no/udoc/trolltech-documentation-process), and one of the lessons from developing that documentation process was that developers make fewer mistakes if they document both values of a boolean in that waffling style.

In making it shorter, you accidentally introduced a mistake. A very easy mistake when shortening.